src/components/autosuggest/_macro.spec.js

```diff
@@ -5,130 +5,177 @@ import * as cheerio from 'cheerio';
 import axe from '../../tests/helpers/axe';
 import { renderComponent, templateFaker } from '../../tests/helpers/rendering';
 
-const EXAMPLE_AUTOSUGGEST = {
-    id: 'country-of-birth',
-    input: {
-        label: {
-            text: 'Current name of country',
-            description: 'Enter your own answer or select from suggestions',
-            id: 'country-of-birth-label',
-            classes: 'extra-label-class',
-        },
-        autocomplete: 'off',
-    },
-    instructions: 'Use up and down keys to navigate.',
-    ariaYouHaveSelected: 'You have selected',
-    ariaMinChars: 'Enter 3 or more characters for suggestions.',
-    minChars: 2,
-    ariaResultsLabel: 'Country suggestions',
-    ariaOneResult: 'There is one suggestion available.',
-    ariaNResults: 'There are {n} suggestions available.',
-    ariaLimitedResults: 'Type more characters to improve your search',
-    moreResults: 'Continue entering to improve suggestions',
-    resultsTitle: 'Suggestions',
-    resultsTitleId: 'country-of-birth-suggestions',
-    autosuggestData: '/examples/data/country-of-birth.json',
-    noResults: 'No suggestions found. You can enter your own answer',
-    typeMore: 'Continue entering to get suggestions',
-};
-
-describe('macro: autosuggest', () => {
-    it('passes jest-axe checks', async () => {
-        const $ = cheerio.load(renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
-    });
+import { EXAMPLE_AUTOSUGGEST } from './_test-examples';
 
-    it('has expected id on container element', () => {
-        const $ = cheerio.load(renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
+describe('FOR: macro: autosuggest', () => {
```

Comment:
Here to match other test refactors this should be `FOR: autosuggest` but I think we need a discussion on how to differentiate this from the js tests